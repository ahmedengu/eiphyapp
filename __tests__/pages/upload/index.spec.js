import { mount } from 'enzyme';
import { axe } from 'jest-axe';
import Upload from '../../../pages/upload/index';

jest.mock('../../../components/header', () => () => <header>Header</header>);

function setup(props = {}) {
  return mount(<Upload {...props} />);
}

describe('Upload Tests', () => {
  it('should work', () => {
    const actual = setup();
    expect(actual).toBe(actual);
  });

  it('should match snapshot', async () => {
    const actual = setup();
    const html = actual.html();
    expect(html).toMatchSnapshot();
    expect(await axe(html)).toHaveNoViolations();
  });
});
