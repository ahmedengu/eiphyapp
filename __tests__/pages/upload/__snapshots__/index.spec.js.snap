// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload Tests should match snapshot 1`] = `
<div id="divLoading"
     class="show"
>
</div>
<div id="layout">
  <header>
    Header
  </header>
  <main class="main-layout">
    <div class="container">
      <div class="row text-center">
        <h2>
          UPLOAD
        </h2>
        <p>
          Upload your GIFs and Stickers to share on Facebook, Twitter, Instagram, text message, email, and everywhere else.
        </p>
      </div>
      <section class="container">
        <div tabindex="0"
             class="dropzone"
        >
          <input accept="image/*"
                 type="file"
                 style="display: none;"
                 autocomplete="off"
                 tabindex="-1"
          >
          <p>
            Select or Drag and Drop your Gifs/Memes
          </p>
        </div>
        <div>
          <h3>
            Files
          </h3>
        </div>
      </section>
    </div>
  </main>
  <footer class="mastfoot mt-auto">
    <div class="inner">
    </div>
  </footer>
</div>
`;
